exchanges/btse/btse_websocket.go

```diff
@@ -360,56 +365,73 @@ func (b *BTSE) orderbookFilter(price, amount float64) bool {
 	return price == 0 || amount == 0
 }
 
-// GenerateDefaultSubscriptions Adds default subscriptions to websocket to be handled by ManageSubscriptions()
-func (b *BTSE) GenerateDefaultSubscriptions() (subscription.List, error) {
-	var channels = []string{"orderBookL2Api:%s_0", "tradeHistory:%s"}
```

Comment:
Can we reinstate orderbookL2?

Reply:
Literally: Nope 😛 
Not working, moved to a new URL.

Reply:
🪦